site/topics/representation/representation.rst

```diff
@@ -7,6 +7,203 @@ Representation
 Genotype vs. Phenotype
 ======================
 
+* As previously discussed
+
+    * The phenotype is a solution to a problem
+    * The genotype is an encoded solution to a problem
+
+
+.. figure:: ../representation/genotype_phenotype_space.png
+    :width: 500 px
+    :align: center
+
+    Visualization of the genotype and phenotype spaces. In this example, the phenotype space consists of integers while
+    the genotype space encodes integers as unsigned binary numbers.
+
+
+* The choice of representation can have a large impact on the performance of the evolutionary computation algorithm
+* Further, the difference between solutions in the phenotype space can differ from the difference in the genotype space
+
+    * For example, consider the unsigned binary number maximization problem
+    * The number :math:`7` is ``0111`` in the genotype space
+    * The number :math:`8` is ``1000`` in the genotype space
+    * The difference in the phenotype space (:math:`8 - 7`) is :math:`1`
+    * The difference in the genotype space, if using *Hamming distance*, is :math:`4`
+
+        * The number of bits that would need to change to get from ``0111`` to ``1000`` is :math:`4`
+
+
+.. figure:: ../representation/hamming_distance_cube.png
+    :width: 333 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Hamming_distance
+
+    Graph representing the Hamming distance between the binary numbers 0 to 7. Each vertex represents a binary number
+    and each edge between vertices represents one unit of distance. The minimum distance between vertices is the
+    Hamming distance between the binary numbers. For example, ``000`` and ``101`` has a minimum distance of two edge
+    between them and have a Hamming distance of two from each other.
+
+
+
+TSP Example
+-----------
+
+* Consider the TSP example previous discussed
+
+.. figure:: ../representation/tsp_horse.png
+    :width: 333 px
+    :align: center
+    :target: https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/pull/64#discussion_r1284677649
```

Comment:
This may not work because it's a private repo. I can send you the SVG if it fails

Reply:
The link is to a public repo 
![image](https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/assets/1328048/5324c3e0-0950-489a-879e-859124d87bcb)

Reply:
Oh. Iunno why I figured this was private. I blame Dave.

Reply:
> I blame Dave

It's the only reasonable explanation.